students/marc_charbo/lesson06/mailroom_v4.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import logging.config
+
+#logging.cong file
+logging.config.fileConfig('logging.conf')
+# create logger
+logger = logging.getLogger('mailRoom')
+
+DONOR_LIST = {'Jim':[25.00, 150.00, 2000.00, 100000.00],'Linda':[10000.25],'Bob':[5.03, 100.01, 6.00]}
+
+def send_letter():
+    """write thank you note to all users in donor list"""
+    for key, value in DONOR_LIST.items():
+        with open(key+'.txt', 'w') as file:
+            file.write('Dear {},\n'.format(key))
+            file.write('\n')
+            file.write('\tThank you for your very kind donation of ${:.2f},\n'.format(sum(value)))
+            file.write('\n')
+            file.write('\tIt will be put to very good use.\n')
+            file.write('\n')
+            file.write('\t\t\tSincerely,\n')
+            file.write('\t\t\t\t-The Team')
+            file.close()
```

Comment:
You might want to just form the string first and then just have one call to `file.write()`